app/create_file.py

```diff
@@ -1 +1,55 @@
-# write your code here
+import sys
+import os
+from datetime import datetime
+
+
+def create_file(path: str, file_name: str) -> None:
+    file_path = os.path.join(path, file_name)
+
+    if not os.path.exists(path):
+        os.makedirs(path)
```

Comment:
You can simplify your code with using `exist_ok =True`
```suggestion
    os.makedirs(path, exist_ok =True)
```